backtrace-library/src/main/cpp/backends/crashpad-backend.cpp

```diff
@@ -120,9 +123,12 @@ bool InitializeCrashpad(jstring url,
 
     // Start crash handler
     client = new crashpad::CrashpadClient();
+    client->EnableCrashLoopDetection();
 
-    initialized = client->StartHandlerAtCrash(handler, db, db, backtraceUrl, attributes,
-                                              arguments);
+    // Get consecutive crashes count BEFORE any handler started,
+    // as it writes extra line into CSV, what leads to getting 0 for each next ConsecutiveCrashesCount call
+    consecutive_crashes_count = crashpad::CrashpadClient::ConsecutiveCrashesCount(db);
```

Comment:
We are using this value later when we call IsSafeModeRequired f.e. from MainActivity. 
From my tests followed that this value should be set before we set up any handler to crashpad, because then I always received 0.